core/templates/pages/learner-dashboard-page/learner-dashboard-page.mainpage.html

```diff
@@ -9,7 +9,20 @@
       if (!window.angular.resumeBootstrap) {
         window.angular.resumeBootstrap = function(extraModules) {};
       }
+
+window.addEventListener('load', function() {
```

Comment:
Do not add scripts here. JavaScript code belongs in the JS file. Also this isn't an "Angular way" of solving this.

I'm closing this PR because this solution is completely off and not consistent with the rest of our codebase.

Reply:
Ok @seanlip .  
Thanks for reviewing.

Reply:
I was also thinking not to add script here but because previous code was having script inside the same HTML that's why I done this.

Reply:
OK, but please note that that code is just boilerplate for setting up Angular, and we shouldn't add to it.